assets/src/dashboard/components/cardGallery/index.js

```diff
@@ -167,7 +167,6 @@ function CardGallery({ story, isRTL, galleryLabel }) {
                 width={miniWrapperSize.width}
               >
                 <MiniCardButton
-                  gridRef={gridRef}
```

Comment:
gridRef is actually assigned at the container - `<MiniCardsContainer>`, this just snuck through and isn't doing anything.